recipes-ni/niconfigpersistentlogs/files/niconfigpersistentlogs

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+set -uo pipefail
+
+error_and_die() {
+set +u
+    if [ -n "$1" ]; then
+        echo >&2 "$1"
+    fi
+set -u
+    echo >&2 "ERROR"
```

Comment:
Why not like: `echo >&2 "ERROR: $1"`?

Reply:
I had been printing the ERROR line separately because in some cases a call to a utility (grep, sed) would directly write its own output to stderr. I realized it would be better to just capture the stderr output of those calls directly and pass them along to error_and_die instead.